src/algebra/ring.lean

```diff
@@ -7,7 +7,6 @@ import algebra.group.hom
 import algebra.group.units
 import tactic.norm_cast
 import tactic.split_ifs
-import algebra.group.units
```

Comment:
Now here's a redundant import removal I can get behind.

Reply:
Sorry that import removal went through too quickly. :-(